app/utils/client/lib/roomTypes.js

```diff
@@ -105,16 +112,16 @@ export const roomTypes = new class RocketChatRoomTypes extends RoomTypesCommon {
 		}
 		return this.roomTypes[roomType].showJoinLink(roomId);
 	}
-	getNotSubscribedTpl(roomId) {
+	getTemplate(roomId, templateName) {
```

Comment:
I didn't like changing this function to make it more generic.. It was made specific on purpose.. I would rather create a new one to get the `readOnlyTpl`..

to avoid duplicating code that finds the room I'd then create a private method/function to do that.

Reply:
Done.